scan-external-contacts/config.yml

```diff
@@ -0,0 +1,144 @@
+---
+title: Scan through External Contacts
+description: See how to scan through external contacts, external organizations, relationships and notes using the External Contacts API.
+categories:
+- Contacts
+steps:
+- title: Introduction
+  content: |
+    This tutorial walks through the steps of implementing a function to scan through the external contacts of a Genesys Cloud organization, and to process them before saving them into a file.
```

Comment:
It might be useful to introduce what external contact is and why you would be interested in using them.  Sometimes I think our tutorials assume a certain level of familiarity with the topics, but you might be dealing with someone completely new and that little bit fo background would help the reader.